build-system/release-tagger/create-release.js

```diff
@@ -23,20 +23,113 @@
  */
 
 const argv = require('minimist')(process.argv.slice(2));
+const dedent = require('dedent');
 const {
   createRelease,
   getPullRequestsBetweenCommits,
   getRelease,
 } = require('./utils');
 const {GraphQlQueryResponseData} = require('@octokit/graphql'); //eslint-disable-line no-unused-vars
 
+/**
+ * Format pull request line
+ * @param {GraphQlQueryResponseData} pr
+ * @return {string}
+ */
+function _formatPullRequestLine(pr) {
+  const {mergeCommit, number, title, url} = pr;
+  const {abbreviatedOid, commitUrl} = mergeCommit;
+  return dedent`\
+  <a href="${commitUrl}"><code>${abbreviatedOid}</code></a> ${title} \
+  (<a href="${url}">#${number}</a>)`;
+}
+
+/**
+ * Create raw notes
+ * @param {Array<GraphQlQueryResponseData>} prs
+ * @return {Array<string>}
+ */
+function _createRawNotes(prs) {
+  return prs.map((pr) => _formatPullRequestLine(pr));
```

Comment:
Looks like you can just use `prs.map(_formatPullRequestLine)`, maybe even instead of having a separate function for this?